labgrid/driver/sshdriver.py

```diff
@@ -164,15 +166,13 @@ def get_status(self):
     @Driver.check_active
     @step(args=['filename', 'remotepath'])
     def put(self, filename, remotepath=''):
-        transfer_cmd = [
-            "scp",
-            self.ssh_prefix,
-            "-P", str(self.networkservice.port),
-            filename,
-            "{user}@{host}:{remotepath}".format(
-                user=self.networkservice.username,
-                host=self.networkservice.address,
-                remotepath=remotepath)
+        transfer_cmd = ["scp"] + self.ssh_prefix + \
+            ["-P", str(self.networkservice.port),
```

Comment:
This works as well:
```
>>> l = [1,2,3]
>>> print([0, *l, 4])
[0, 1, 2, 3, 4]
```
So use
```suggestion
        transfer_cmd = [
            "scp",
            *self.ssh_prefix,
            "-P", str(self.networkservice.port),
```
and avoid the reformatting.